upstream-community-operators/apicurio-registry/0.0.1/apicurio-registry-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: apicurio-registry.v0.0.1
+  namespace: placeholder
+  annotations:
+    alm-examples: >-
+      [{"apiVersion":"apicur.io/v1alpha1","kind":"ApicurioRegistry","metadata":{"name":"example-apicurioregistry"},"spec":{"image":{"registry":"docker.io/apicurio","version":"latest-release"},"configuration":{"persistence":"mem"},"deployment":{"route":"registry.example.com"}}}]
+    categories: ' Streaming & Messaging'
+    certified: 'false'
+    createdAt: '2020-05-27'
+    description: Deploy and manage Apicurio Registry on Kubernetes.
+    containerImage: 'apicurio/apicurio-registry-operator:0.0.1'
+    support: 'Apicurio Project'
+    capabilities: Basic Install
+    repository: 'https://github.com/Apicurio/apicurio-registry-operator'
+spec:
+  displayName: Apicurio Registry Operator
+  description: >
+    ## Apicurio Registry
+
+
+    Apicurio Registry stores and retrieves API designs and event schemas, and
+    gives you control of their evolution.
+
+
+    **Features**
+
+    - Supports: Apache Avro, AsyncAPI, GraphQL, JSON Schema, Kafka Connect
+    Schema, OpenAPI, Protobuf
+
+    - Provides a REST API to manage the artifacts and artifact meta-data
+
+    - Includes Serializers and Deserializers for Kafka client integration
+
+    - Configurable rules to control schema validity and evolution
+    (compatibility)
+
+    - Storage options: Kafka Streams, Kafka, PostgreSQL, in-memory
+
+    - Compatible with Confluent and IBM APIs
+
+    - Runs on a lightweight Quarkus platform
+
+    - Includes Maven plugin to integrate with Maven based builds
+
+    ## Apicurio Registry Operator
+
+
+    Provides a quick and easy way to deploy and manage an Apicurio Registry on
+    Kubernetes.
+
+
+    **Features**
+
+    - Supports basic Install and configuration of the Registry
+
+    - Can optionally create an Ingress to access the API and UI  on HTTP port 80
+
+    - Manual horizontal scaling
+
+    - Easily perform a rolling upgrade of the Registry
+
+    ## Prerequisites
+
+    This operator does not deploy storage for the Registry. Therefore, some storage options require that the chosen persistence service is already set up. You can do this using an operator for the specified service, such as Strimzi for Kafka Streams.
+  maturity: alpha
+  version: 0.0.1
+  replaces: ''
```

Comment:
Since it is first version, please remove `replaces: ''`. Please use it only when you are upgrading to new version. When you will fix it, i will merge it.

Reply:
Fixed and squashed!